src/EPR.Calculator.API.Data/bin/Debug/net8.0/EPR.Calculator.API.Data.runtimeconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "runtimeOptions": {
```

Comment:
The bin directory can be added to .gitignore.

Reply:
Done